src/features/Pokedex/Search/Search.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { Formik, Field } from 'formik';
+import { object, string } from 'yup';
+
+function InnerForm({ errors, handleSubmit, isSubmitting, touched }) {
```

Comment:
This form is wrapped by the `<Formik />` component, and these props are essentially the `formik` API. (There are a lot more props to use btw).

What this allows us to do, is be able to export this SFC and test the UI and shallow user interactions, which are way easier to do than testing the entire form logic.

We can leverage functional tests to actually ensure that `formik` integrates with our logic and that it works as expected, to cover the gaps that unit specs don't fill.